### DISCUSSION TITLE
Internal Timer to Track Hours Worked

### DISCUSSION THREAD

**Discussion by njs811271** [godotengine/godot-proposals#6671] [Editor]
I would like a timer integrated into the Godot project file. 

When I complete a game I want to be able to determine how much time I've devoted to programming it. This will allow me to determine what I'm paying myself per hour, and help monitor my involvement with a project. 

It could be displayed conveniently on the project list filename at startup.

**Comment by AThousandShips** [comment]
Since this is a feature that can easily be achieved with third party tools and is very specific I don't think it belongs in the engine itself

**Reply by njs811271** [reply]
That's disappointing. I would think this would be interesting to everyone that used the engine. If for nothing other than to tell their friends how long they worked on something.

I'm new to Godot, how could I write something that integrates this?

**Reply by dalexeev** [reply]
See [Editor plugins](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/index.html). Also, you can check out the Asset Library, as far as I remember there are already such plugins.
